code/game/objects/items/storage/backpack.dm

```diff
@@ -50,28 +50,30 @@
 	max_w_class = 4
 	max_storage_space = 28
 
+///Checks whether or not we fail for this interaction and handles failing if we do
 /obj/item/storage/backpack/holding/proc/failcheck(mob/user)
-	if (prob(reliability)) 
+	if(prob(reliability)) 
 		return TRUE //No failure
-	if (prob(reliability))
+	if(prob(reliability))
 		to_chat(user, "<span class='warning'>The Bluespace portal resists your attempt to add another item.</span>")
 	else
 		to_chat(user, "<span class='warning'>The Bluespace generator malfunctions!</span>")
-		for (var/obj/O in src.contents) //it broke, delete what was in it
-			qdel(O)
-		crit_fail = 1
+		for (var/obj/content in src.contents) //it broke, delete what was in it
```

Comment:
AS in

Reply:
I remembered we have a macro explicity for qdel'ing a list